### PR TITLE
added blurb about font-display and cleaned up language format

### PR DESCRIPTION
**Pull request by CrystalOnScript**
Resolves [issue #27214](https://github.com/ampproject/amphtml/issues/27214)

### CODE REVIEW

File: extensions/amp-font/amp-font.md [hunk 1]
```diff
@@ -26,9 +26,16 @@ limitations under the License.
 
 # amp-font
 
-## Behavior
+## Do not use
+
+Use the [`font-display`](https://developer.mozilla.org/en-US/docs/Web/CSS/@font-face/font-display) descriptor to specify how to display a font face dependent on whether and when it is ready.
```
Comment (sebastianbenz) [inline extensions/amp-font/amp-font.md]: Can you provide a code snippet explaining how to cover the amp-font use cases with font-display?

File: extensions/amp-font/amp-font.md [hunk 2]
```diff
@@ -26,9 +26,16 @@ limitations under the License.
...
 # amp-font
 
-## Behavior
+## Do not use
+
+Use the [`font-display`](https://developer.mozilla.org/en-US/docs/Web/CSS/@font-face/font-display) descriptor to specify how to display a font face dependent on whether and when it is ready. 
+
+The `amp-font` component was AMP's solution to controlling timeouts on font loading. With the wide browser adoption of font-display, this component is no longer necessary. It is here for legacy users. 
+
+### Usage
```
Comment (sebastianbenz) [inline extensions/amp-font/amp-font.md]: Suggestion: remove the old documentation.

File: extensions/amp-font/amp-font.md [hunk 3]
```diff
@@ -26,9 +26,16 @@ limitations under the License.
 
 # amp-font
```
Comment (sebastianbenz) [inline extensions/amp-font/amp-font.md]: Change to `amp-font (deprecated)`

File: extensions/amp-font/amp-font.md [hunk 4]
```diff
@@ -26,9 +26,16 @@ limitations under the License.
...
 
-## Behavior
+## Do not use
```
Comment (sebastianbenz) [inline extensions/amp-font/amp-font.md]: Replace with a sentence: "This component has been deprecated and should no longer be used. Instead, ..."

**Comment by CrystalOnScript** [comment]
@nainar, yup, no new information - but I couldn't not clean it up...

File: extensions/amp-font/amp-font.md (hunk 2 above)
Comment (CrystalOnScript) [reply extensions/amp-font/amp-font.md]: I'm torn about removing it completely. 137 million pages might still need it...

File: extensions/amp-font/amp-font.md (hunk 1 above)
Comment (CrystalOnScript) [reply extensions/amp-font/amp-font.md]: I'll look into it. @nainar @caroqliu do either of you have one/know someone who might have one ready to go?

Comment (caroqliu) [reply extensions/amp-font/amp-font.md]: I don't unfortunately :(

File: extensions/amp-font/amp-font.md [hunk 5]
```diff
@@ -24,11 +24,16 @@ See the License for the specific language governing permissions and
...
+# amp-font (deprecated)
 
-## Behavior
+Replace with a sentence: "This component has been deprecated and should no longer be used. Instead, use the [`font-display`](https://developer.mozilla.org/en-US/docs/Web/CSS/@font-face/font-display) descriptor to specify how to display a font face dependent on whether and when it is ready.
+
+The `amp-font` component was AMP's solution to controlling timeouts on font loading. With the wide browser adoption of font-display, this component is no longer necessary. It is here for legacy users.
+
+## Usage
+
+Historically, the `amp-font` extension was used for controlling timeouts on font loading. **We recommend implementing this functionality with [`font-display`](https://developer.mozilla.org/en-US/docs/Web/CSS/@font-face/font-display) instead.**
```
Comment (mrjoro) [inline extensions/amp-font/amp-font.md:36]:
```suggestion
```

File: extensions/amp-font/amp-font.md [hunk 6]
```diff
@@ -52,54 +57,52 @@ Example:
...
+Behaves same as standard CSS [`font-weight` property](https://developer.mozilla.org/en-US/docs/Web/CSS/font-weight).
+
+### `font-style` (optional)
+
+Behaves same as standard CSS [`font-style` property](https://developer.mozilla.org/en-US/docs/Web/CSS/font-style).
+
+### `font-variant` (optional)
+
+Behaves same as standard CSS [`font-variant` property](https://developer.mozilla.org/en-US/docs/Web/CSS/font-variant).
+
```
Comment (mrjoro) [inline extensions/amp-font/amp-font.md:101]:
```suggestion
```

**Pull request merged**